data/calculated_metrics_testfile.json

```diff
@@ -0,0 +1,3 @@
+{
+    "Keep this file empty": []
```

Comment:
Why do we need 7 different json files to be in master? I believe that it was recommended by the course instructor to have a tmpdir for any test files to be modified/read exclusively in the test environment. This type of thing can be accomplished with Pytest fixtures, as shown in a tutorial [here](https://code-maven.com/pytest-fixtures-temporary-directory)

Reply:
Thank you for putting a link to the tutorial. These files exist for now to make sure the test cases are working correctly. Considering that this is a work in progress, I will be adding the fixtures you suggested soon.

Reply:
This can be resolved later.